src/js/modules/infragistics.ui.editors.js

```diff
@@ -4751,18 +4723,15 @@
 				}
 
 				// A.M. April 12, 2017 #852 Don't allow groupSeparator and groupSeparator to use the same symbol
-				if (this.options[ option ] === this.options.decimalSeparator) {
+				if (this.options[ option ] === this._getOptionOrRegionalValue("decimalSeparator")) {
 					throw new Error(this._getLocaleValue("decimalSeparatorEqualsGroupSeparatorErrorMsg"));
 				}
 				break;
-			case "regional":
-				this.options[ option ] = prevValue;
-				throw new Error(this._getLocaleValue("setOptionError") + option);
 
 			case "excludeKeys":
 			case "includeKeys":
 				this.options[ option ] = prevValue;
-				throw new Error(this._getLocaleValue("numericEditorNoSuchOption"));
+				throw new Error(this._getLocaleValue("setOptionError") + option);

```

Comment:
I don't think this should be changing though, as the numerics intentionally throw a different error (since they don't expose those options at all)